dashed/models.py

```diff
@@ -634,13 +634,14 @@ def fetch_metadata(self):
             db.session.flush()
             if not dbcol:
                 dbcol = TableColumn(column_name=col.name)
-
+                num_types = ('DOUBLE', 'FLOAT', 'INT', 'BIGINT')
```

Comment:
Good catch. The bug I fixed here with mysql was very confusing where the type was `BIGINT(64)` instead of just `BIGINT`